modules/dhcp_common/record/reservation.rb

```diff
@@ -7,7 +7,7 @@ class Reservation < Record
     def initialize(name, ip_address, mac_address, subnet, options = {})
       @type = "reservation"
       @name = name
-      super(ip_address, mac_address, subnet, options)
+      super(ip_address, mac_address, subnet, {:deleteable => true}.merge(options))
```

Comment:
I don't know why but this line created regression from 1.16 which I reported here: http://projects.theforeman.org/issues/21975